include/termcolor/termcolor.hpp

```diff
@@ -45,14 +45,14 @@
 
 namespace termcolor
 {
-    // Forward declaration of the `__internal` namespace.
+    // Forward declaration of the `_internal` namespace.
     // All comments are below.
-    namespace __internal
+    namespace _internal
```

Comment:
Usually I don't like it since it makes noise in the current commit. Hence, it complicates patch review - I need to focus on both things at once:

* macro renaming
* namespace renaming

It's better to make those renaming in separate commits and pull requests. Besides, I'd questioned the idea of renaming that namespace.

@nabijaczleweli so please, rename only macros - that makes pretty sense to me. And if you think that namespace must be renamed either - let's discuss it in separate pull request/issue.